packages/gasket-plugin-config/lib/middleware.js

```diff
@@ -16,6 +16,16 @@ function handler(gasket) {
         req,
         res);
 
+      // Error handling
+      // Better error description for the appRequestConfig lifecycles
+      // In the event a config object isn't returned from the hook(s)
+      // it will result in a error in the destructuring of req.config
+      if (!req.config) {
+        throw new Error(
```

Comment:
Should we use the `gasket.logger` object here?

Reply:
I think a hard throw on these here makes sense. It should fail the app.